admin-dev/themes/new-theme/js/pages/product/components/combination-list/combinations-list-renderer.ts

```diff
@@ -276,7 +277,15 @@ export default class CombinationsListRenderer {
     const productPrice = this.productFormModel.getPriceTaxExcluded();
     const $finalPrice = $(CombinationsMap.list.finalPrice, $row);
     const $finalPricePreview = $finalPrice.siblings(CombinationsMap.list.finalPricePreview);
-    const finalPrice = this.productFormModel.displayPrice(productPrice.plus(priceImpactTaxExcluded));
+    let combinationPrice = productPrice.plus(priceImpactTaxExcluded);
+
+    const ecoTax = $(ProductPriceMap.ecoTaxTaxExcluded).val();
+
+    if (ecoTax) {
+      combinationPrice = combinationPrice.plus(new BigNumber(ecoTax.toString()));
+    }
```

Comment:
To avoid invalid computing (just in case) maybe the BigNumber value should be stored and we should check if it is not NaN (this happened too many times for me to ignore it 😅) Besides the combination list component already has the `productModel` in its dependencies so you can reuse it (it avoids adding a new form mapping which is already handled by the model itself)

Reply:
```suggestion
    const ecoTax = this. productModel.getBigNumber('price.ecotaxTaxExcluded') ?? new BigNumber(0);

    if (!ecoTax.isNaN()) {
      combinationPrice = combinationPrice.plus(ecoTax);
    }
```